animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -112,6 +116,9 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
             self.newInput("Float", "End Size", "endSize", value = 0.5, minValue = 0)
             self.newInput("Float", "Start Angle", "startAngle", value = 0)
             self.newInput("Float", "End Angle", "endAngel", value = 6 * PI)
+            self.newInput("Float", "Altitude", "altitude", value = 0)
```

Comment:
Let us call this Height to be consistent with other modes.

Reply:
acturally, at the begining i  used the "Height", but i found it overlay the grid mode Height, and can not set default value to 0, i don't know why.

Reply:
In this case, you can just change the identifier to `spiralHeight` (Third argument).